### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sophia-guo**
Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,3 +115,8 @@ THe source of test against JDK. Default is `upstream`. Supported value is [`upst
   - install-jdk: JDK installed by [AdoptOpenJDK/install-jdk](https://github.com/AdoptOpenJDK/install-jdk) | [actions/setup-java](https://github.com/actions/setup-java)
   - github-hosted : pre-installed JDK on github-hosted environment
 
+### openjdk_testRepo
+Openjdk-test git repo, the AQA test. Default is `openjdk-tests:master`. Support developer's personal repo. 
+
+### tkg_Repo
+TKG git repo, the AQA underlining test framework. Support developer's personal repo.
```
Comment (smlambert) [inline README.md:122]: Reword to: 
TKG git repo, the underlying framework for the AQA test suite.  Parameter can be set to use developer's personal repo.

File: README.md [hunk 2]
```diff
@@ -114,3 +115,8 @@ THe source of test against JDK. Default is `upstream`. Supported value is [`upst
   - install-jdk: JDK installed by [AdoptOpenJDK/install-jdk](https://github.com/AdoptOpenJDK/install-jdk) | [actions/setup-java](https://github.com/actions/setup-java)
   - github-hosted : pre-installed JDK on github-hosted environment
 
+### openjdk_testRepo
+Openjdk-test git repo, the AQA test. Default is `openjdk-tests:master`. Support developer's personal repo.
```
Comment (smlambert) [inline README.md:119]: openjdk-tests git repo, that holds the definitions for the AQA test suite.  Parameter can be set to use developer's personal repo.

Comment (sophia-guo) [reply README.md]: Updated. Thanks!

File: README.md (hunk 1 above)
Comment (sophia-guo) [reply README.md]: Updated. Thanks!

**Review by smlambert — APPROVED** [review]
LGTM 👍

**Pull request merged**